src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -96,12 +96,12 @@ private static Person createEditedPerson(Person personToEdit, EditPersonDescript
         assert personToEdit != null;
 
         Name updatedName = editPersonDescriptor.getName().orElse(personToEdit.getName());
-        Phone updatedPhone = editPersonDescriptor.getPhone().orElse(personToEdit.getPhone());
-        Email updatedEmail = editPersonDescriptor.getEmail().orElse(personToEdit.getEmail());
-        Address updatedAddress = editPersonDescriptor.getAddress().orElse(personToEdit.getAddress());
-        Set<Tag> updatedTags = editPersonDescriptor.getTags().orElse(personToEdit.getTags());
+//        Phone updatedPhone = editPersonDescriptor.getPhone().orElse(personToEdit.getPhone());
+//        Email updatedEmail = editPersonDescriptor.getEmail().orElse(personToEdit.getEmail());
+//        Address updatedAddress = editPersonDescriptor.getAddress().orElse(personToEdit.getAddress());
+//        Set<Tag> updatedTags = editPersonDescriptor.getTags().orElse(personToEdit.getTags());
```

Comment:
I think you have to either delete the commented part or do the indentations probably because it is failing the checks